apps/browser/src/vault/popup/components/fido2/fido2-cipher-row.component.ts

```diff
@@ -17,4 +17,14 @@ export class Fido2CipherRowComponent {
   protected selectCipher(c: CipherView) {
     this.onSelected.emit(c);
   }
+
+  protected getSubName(c: CipherView): string {
```

Comment:
🎨 Nit: We can make this more readable by eliminating the nested ifs. Not a blocker, just a suggestion

```suggestion
  protected getSubName(c: CipherView): string | null {
    const fido2Credentials = c.login?.fido2Credentials;
    
    if (!fido2Credentials || fido2Credentials.length === 0) {
        return null;
    }
    
    const [fido2Credential] = fido2Credentials;
    
    return c.name !== fido2Credential.rpId ? fido2Credential.rpId : null;
}
```